### PR TITLE
Timer top model status update

### PR DESCRIPTION
**Pull request by pasnox**
Add a way to update a timer state change while it did not triggered a timeout recently.

### CODE REVIEW

**Comment by pasnox** [comment]
This covers both QTimer and raw timers.
I will check for QQmlTimer support now, probably using something like runningChanged signal as this class does not expose a timer id.

**Comment by pasnox** [comment]
QQmlTimer support added

File: plugins/timertop/clienttimermodel.cpp [hunk 1]
```diff
@@ -44,24 +48,60 @@ ClientTimerModel::~ClientTimerModel()
...
+            const TimerId::Type type = TimerId::Type(typeSibling.data(TimerModel::TimerTypeRole).toInt());
+            QFont font = QApplication::font("QAbstractItemView");
+            font.setStrikeOut(type == TimerId::InvalidType || state == TimerIdInfo::InvalidState);
+            return QVariant::fromValue(font);
+        } else if(role == Qt::BackgroundRole) {
+            const QModelIndex sibling = index.sibling(index.row(), TimerModel::ObjectNameColumn);
+            const TimerId::Type type = TimerId::Type(sibling.data(TimerModel::TimerTypeRole).toInt());
+            switch (type) {
+            case TimerId::InvalidType:
+                return QColor(255, 0, 0, 80);
```
Comment (vkrause) [inline plugins/timertop/clienttimermodel.cpp]: Do these hardcoded colors work with light and dark color schemes?

Comment (pasnox) [reply plugins/timertop/clienttimermodel.cpp]: Tested on both, looks ok.

**Comment by pasnox** [comment]
@volker ?

**Pull request merged**